auth.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/xmidt-org/arrange"
+	"github.com/xmidt-org/bascule/basculechecks"
+	"github.com/xmidt-org/bascule/basculehttp"
+	"go.uber.org/fx"
+)
+
+func provideAuth(configKey string) fx.Option {
+	return fx.Options(
```

Comment:
I'm not sure, but I think we can also call `basculehttp.ProvideLogger()`?
https://github.com/xmidt-org/bascule/blob/afed689c984f8de4b99b4a1297733a60978b380f/basculehttp/log.go#L86
I'm not sure the current logger state for authbaton - I just saw that the old go-kit logger is still being provided for something, but it's not needed for bascule.

Reply:
`ProvideServerChain` already calls `ProvideLogger` https://github.com/xmidt-org/bascule/blob/c011b128d6b95fa8358228535c63d1945347adaa/basculehttp/chain.go#L49 but I just realized that we don't really need `gokitLogger` 🎉